.dependabot/config.yml

```
@@ -5,3 +5,15 @@ update_configs:
   - package_manager: "terraform"
     directory: "/"
     update_schedule: "daily"
+
```

we should probably add `go.dep` and `go.sum` here too
```suggestion

  - package_manager: "go:dep"
    directory: "/tests"
    update_schedule: "daily"

  - package_manager: "go:modules"
    directory: "/tests"
    update_schedule: "daily"
```

---

Fixed

---

We're not actually using `go:dep`, that's the one we replaced with go modules. Doesn't hurt to be here, just fyi.

---

I only added the configuration for `go:modules`